backend/api/submissions/permissions.py

```diff
@@ -27,12 +28,10 @@ def has_permission(self, source, info):
         if user_info.is_staff or source.speaker_id == user_info.id:
             return True
 
-        if Submission.objects.filter(
-            speaker_id=user_info.id, conference=conference
-        ).exists():
-            return True
+        if conference.is_voting_closed:
```

Comment:
Moved this here below because we don't want the fields to be visible if the talk was not accepted, and if the user could have seen the talk (because, admin or author) should be already fall into one of the conditions above